entity-management-zoho/src/main/java/eu/europeana/entitymanagement/zoho/organization/ZohoOrganizationConverter.java

```diff
@@ -0,0 +1,78 @@
+package eu.europeana.entitymanagement.zoho.organization;
+
+import static eu.europeana.entitymanagement.zoho.utils.ZohoUtils.toIsoLanguage;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import com.zoho.crm.api.record.Record;
+
+import eu.europeana.entitymanagement.common.config.AppConfigConstants;
+import eu.europeana.entitymanagement.definitions.model.Address;
+import eu.europeana.entitymanagement.definitions.model.Organization;
+import eu.europeana.entitymanagement.zoho.utils.ZohoConstants;
+import eu.europeana.entitymanagement.zoho.utils.ZohoUtils;
+
+@Component(AppConfigConstants.BEAN_ZOHO_ORGANIZATION_CONVERTER)
```

Comment:
This class will be used only by DereferenceService, we don't need to define it as a bean. It can be manually initialized within the service

Reply:
Improved.